src/pal/src/numa/numa.cpp

```diff
@@ -143,7 +145,8 @@ NUMASupportInitialize()
         struct bitmask *mask = numa_allocate_cpumask();
         int numaNodesCount = numa_max_node() + 1;
 
-        g_cpuCount = numa_num_possible_cpus();
+        g_possibleCpuCount = numa_num_possible_cpus();
+        g_cpuCount = PAL_GetLogicalCpuCountFromOS();
```

Comment:
Can you please change the g_cpuCount computation so that it is initialized to zero here and nodeCpuCount is added to that in the loop below? While I assume it would get the same value, I would prefer getting all the information from the numa here for consistency reasons.